NEWS.rst

```diff
@@ -9,6 +9,35 @@
 
 .. towncrier release notes start
 
+24.3 (2024-10-26)
+=================
+
+Deprecations and Removals
+-------------------------
+
+- Deprecate wheel filenames that are not compliant with :pep:`440`. (`#12918 <https://github.com/pypa/pip/issues/12918>`_)
+
+Features
+--------
+
+- Detect recursively referencing requirements files and help users identify
+  the source. (`#12653 <https://github.com/pypa/pip/issues/12653>`_)
+- Support for :pep:`730` iOS wheels. (`#12961 <https://github.com/pypa/pip/issues/12961>`_)
+
+Bug Fixes
```

Comment:
even though the problem was not in pip code, but in truststore, maybe it is at least worth mentioning that pip 24.3 now works again on macOS <= 10.13 (which was broken in pip 24.2 release).

Reply:
Good point, done.